packages/gestalt-charts/src/Chart.js

```diff
@@ -1,23 +1,402 @@
-import { type Node } from 'react';
-import { Text } from 'gestalt';
+// @flow strict-local
+import { Fragment, type Node, useMemo } from 'react';
+import {
+  BarChart,
+  CartesianGrid,
+  ComposedChart,
+  Legend,
+  LineChart,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from 'recharts';
+import { Box, Flex, Heading, IconButton, Text, useColorScheme, useDefaultLabel } from 'gestalt';
+import { ChartProvider } from './Chart/ChartContext.js';
+import EmptyBox from './Chart/EmptyBox.js';
+import LegendIcon from './Chart/LegendIcon.js';
+import renderCustomTooltip from './Chart/renderCustomTooltip.js';
+import renderDefaultLegend from './Chart/renderDefaultLegend.js';
+import renderDefaultTooltip from './Chart/renderDefaultTooltip.js';
+import renderElements from './Chart/renderElements.js';
+import renderReferenceAreas from './Chart/renderReferenceAreas.js';
+import usePatterns, { useHexColor } from './Chart/usePatterns.js';
 
-export type Props = {|
-  id?: string,
+type Props = {|
+  accessibilityLabel: string,
+  /**
+   * Prop description.
+   */
+  accessibilitySelected: boolean,
+  /**
+   * Prop description.
+   */
+  biaxial?: 'xAxis' | 'yAxis',
+  /**
+   * Prop description.
+   */
+  data: $ReadOnlyArray<{|
+    name: string | number,
+    [string]: number,
+  |}>,
+  /**
+   * Prop description.
+   */
+  description?: string,
+  /**
+   * Prop description.
+   */
+  domain?:
+    | [
+        number | 'auto' | 'dataMin' | 'dataMax' | ((number) => number),
+        number | 'auto' | 'dataMin' | 'dataMax' | ((number) => number),
+      ]
+    | {|
+        xAxisBottom?: [
+          number | 'auto' | 'dataMin' | 'dataMax' | ((number) => number),
+          number | 'auto' | 'dataMin' | 'dataMax' | ((number) => number),
+        ],
+        xAxisTop?: [
+          number | 'auto' | 'dataMin' | 'dataMax' | ((number) => number),
+          number | 'auto' | 'dataMin' | 'dataMax' | ((number) => number),
+        ],
+
+        yAxisLeft?: [
+          number | 'auto' | 'dataMin' | 'dataMax' | ((number) => number),
+          number | 'auto' | 'dataMin' | 'dataMax' | ((number) => number),
+        ],
+
+        yAxisRight?: [
+          number | 'auto' | 'dataMin' | 'dataMax' | ((number) => number),
+          number | 'auto' | 'dataMin' | 'dataMax' | ((number) => number),
+        ],
+      |},
+  /**
+   * Prop description.
+   */
+  elements: $ReadOnlyArray<{|
+    type: 'line' | 'bar',
+    axis?: 'left' | 'right' | 'bottom' | 'top',
+    id: string,
+    color: '01' | '02' | '03' | '04' | '05' | '06' | '07' | '08' | '09' | '10' | '11' | '12',
+    stackedId?: string,
+    precision?: 'exact' | 'estimate',
+  |}>,
+  /**
+   * Prop description.
+   */
+  height?: number | string,
+
+  /**
+   * Prop description.
+   */
+  layout?: 'horizontal' | 'vertical',
+  /**
+   * Prop description.
+   */
+  legend?: 'auto' | 'hidden' | 'complete',
+  /**
+   * Prop description.
+   */
+  onChangeAccessibility: () => void,
```

Comment:
Is this the accessibility mode on the top? 

Is the only thing this does is change the pattern? Wondering if can not confuse it with actual aria modes, and change it to something more explicit like `onFillPatternChange()` or `onVisualPatternChange()`

Reply:
DONE